ruling/snapshots/no-one-iteration-loop

```diff
@@ -1,2 +1,7 @@
+src/react-native/Libraries/Renderer/ReactFabric-dev.js: 12235
```

Comment:
@stas-vilchik these are not generated? kind of biggy

Reply:
I'm not sure